docs/developer/app-store/apps/adyen/architecture.mdx

```diff
@@ -32,3 +38,26 @@ If Adyen surpasses this limit, the App will return a FAILURE status with an appr
 The app restricts Saleor API response time to 5 seconds for [`TransactionInitializeSession`](developer/extending/webhooks/synchronous-events/transaction.mdx#initialize-transaction-session) and [`TransactionProcessSession`](developer/extending/webhooks/synchronous-events/transaction.mdx#process-transaction-session) subscriptions. If Saleor API surpasses this limit, the App will gracefully continue processing.
 
 If such timeout happens, the created [`TransactionItem`](api-reference/payments/objects/transaction-item.mdx) will not have the metadata from Adyen on `additionalDetails` object, which includes payment method type, credit card brand, etc.
+
+### Granted refunds vs manual refunds
+
+There are differences in how refunds are handled depending on wether an [`OrderGrantedRefund`](/api-reference/orders/objects/order-granted-refund) was created in Saleor and a refund via [`transactionRequestRefundForGrantedRefund`](api-reference/payments/mutations/transaction-request-refund-for-granted-refund) mutation was requested, or a "manual refund" sent via [`transactionRequestAction`](/api-reference/payments/mutations/transaction-request-action) mutation was requested. The differences is in how [`lineItems`](https://docs.adyen.com/api-explorer/Checkout/71/post/payments/_paymentPspReference_/refunds#request-lineItems) are reported to Adyen
+
+#### Manual refund
+
+A manual refund is initiated using the [`transactionRequestAction`](/api-reference/payments/mutations/transaction-request-action) mutation.
+
+When manual refund is requested, the app will send `lineItems` to Adyen when the `amount` requested for refund is equal to or greater than Order's total gross amount.
```

Comment:
```suggestion
When manual refund is requested, the app will send `lineItems` to Adyen for the `amount` equal to or greater than Order's total gross amount.
```

I am not sure if I understand this sentence, but I tried fixing it the way I understand it 😅

Reply:
`amount` is input in the `transactionRequestAction` mutation, so **if** amount == `order.totalGrossAmount` then we send `lineItems`